etc/httpd/global.conf

```diff
@@ -760,3 +760,6 @@ RewriteRule ^(/(?:\w{2,3}(?:-\w{2})?/)?firefox/beta/feedbackprivacypolicy/?)$ /b
 
 # bug 1149019
 RewriteRule ^(/certs/.*)$ /b$1 [PT]
+
+# bug 1252332
+RewriteRule ^(/(?:\w{2,3}(?:-\w{2})?/)?sync/?)$ /b$1 [PT]
```

Comment:
This is slightly different to the pass-through that @jpetto suggested (which also works), but it looks like a lot of re-write rules are now following this format.

@pmclanahan is this now the preferred format for adding a pass through in `global.conf`?